src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,15 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private final Random randomForColor;
+
+    public ColorSupplier() {
+        this.randomForColor = new Random();
+    }
+
+    public Colors getRandomColor() {
+        return Colors.values()[randomForColor.nextInt(Colors.values().length)];
```

Comment:
Let's create an `randomIndex` as a variable.
```suggestion
        int randomIndex = randomForColor.nextInt(Colors.values().length);
        return Colors.values()[randomIndex];
```